sharktank/sharktank/layers/configs/llm_configs.py

```diff
@@ -162,3 +113,62 @@ def _optional_int_prop(p: dict[str, Any], name: str, default_value: int) -> int:
         return int(value)
     except ValueError as e:
         raise ValueError(f"Property '{name}' expected to be an int and was not") from e
+
+
```

Comment:
Adding  LlamaModelConfig to config file make sense, but having create_kv_cache() function in a config file feels odd. Is there a better way to refactor this?

Reply:
I moved `create_kv_cache` function to utils folder, as that seemed to be the best place for it.